### ISSUE TITLE
%GetHeapUsage removed

### ISSUE THREAD

**Issue by jscheid** [NathanaelA/v8-Natives#15]
It looks like `%GetHeapUsage` is no longer available in Chrome 91, which is unfortunate since the code is using it to test for `--allow-natives-syntax` support.

I can't find anything official on when it was removed or why, but:

<img width="577" alt="Screen Shot 2021-05-26 at 20 39 54" src="https://user-images.githubusercontent.com/3514/119629883-8c9b6a00-be62-11eb-9237-61ba0feff0cf.png">

**Comment by NathanaelA** [comment]
I will attempt to look into if they renamed it, cause sometimes they change the names.   Thanks for pinging me on it.

**Comment by NathanaelA** [comment]
I went at looked at the source code, and it was removed at the end of Dec 2020.   I've pinged the developer that removed it, to see if there is a different options as looking thru the new commands I don't see anything equivalent or even related.  :(

**Comment by NathanaelA** [comment]
Unfortunately the developer that removed this, basically stated anything internal (i.e. %xxxx) are removed at whim when no longer used by the code base.   So it is gone for good.

I found that `window.performance.memory` in chrome will give you the Heap / Memory info.  (If you use `--enable-precise-memory-info` you will get more accurate information)

And with Node you can use:
   `process.memoryUsage();`

I'll get this removed from the v8-natives codebase...   With a comment about this...

**Comment by jscheid** [comment]
Sorry, I don't think I explained the problem very well, let me try again:

I don't actually need `%GetHeapUsage` myself at all, but it's what your package is using to detect whether native support is enabled: https://github.com/NathanaelA/v8-Natives/blob/e1e1b2927621342dcb1e14ddde421c21e95e5d34/lib/v8-browser-all.js#L32-L42

So, with `%GetHeapUsage` removed, `testIfAllowNativeSyntax` will always return false (on recent Chrome) and make your package fall back to the dummy implementations... pretty much rendering the whole package useless.

I would suggest that you replace that check either with something "guaranteed" (if there is such a thing) to be around for the foreseeable future, or -- maybe better -- check a list of functions and return `true` for `testIfAllowNativeSyntax` as long as at least one of the functions is available. I suppose ideally you would check all functions used by your package and then emit warnings or disable functionality accordingly, but who has time for that.

(Although I won't need them, thanks for researching and digging up those alternative suggestions for me!)

**Comment by NathanaelA** [comment]
I've pushed a new version that removes this and uses a different test command.   Thanks for letting me know.

**Comment by jscheid** [comment]
Awesome, thanks for the quick turnaround!
